py/desisim/scripts/quickquasars.py

```diff
@@ -717,10 +721,10 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
         specsim_config_file = 'desi'
 
     ### use Poisson = False to get reproducible results.
-    sim_spectra(qso_wave,qso_flux, args.program, obsconditions=obsconditions,spectra_filename=ofilename,
-        sourcetype="qso", skyerr=args.skyerr,ra=metadata["RA"],dec=metadata["DEC"],targetid=targetid,
-        meta=specmeta,seed=seed,fibermap_columns=fibermap_columns,use_poisson=False,
-        specsim_config_file=specsim_config_file, dwave_out=dwave_out)
+    resolution=sim_spectra(qso_wave,qso_flux, args.program, obsconditions=obsconditions,spectra_filename=ofilename,
```

Comment:
Hi @alxogm - these lines are not well indented, it looks like these are four lines instead of a very long one.